metadata/dashboard/sqlLite.py

```diff
@@ -158,6 +158,7 @@ def createTables(self):
             variant     text,
             status      text,
             definition  text,
+            source_text text,
```

Comment:
is it okay to just store the source text? What if we just unpickle the definition binary?

Reply:
This is the question I had reading through the PR: is there a reason we're saving the source text separately? Is that simply to avoid having to unpickle each time we make a call to sources in the dashboard?

Reply:
Because you're not pickling the function as a string, you pickle its logic. 
If I have :
```
def my_function(something):
    return something
```
and pickle it and print the serialized version, I have:
`b'\x80\x04\x95\xf2\x00\x00\x00\x00\x00\x00\x00\x8c\ndill._dill\x94\x8c\x10_create_function\x94\x93\x94(h\x00\x8c\x0c_create_code\x94\x93\x94(C\x02\x02\x01\x94K\x01K\x00K\x00K\x01K\x01K\x03C\x06\x97\x00|\x00S\x00\x94N\x85\x94)\x8c\tsomething\x94\x85\x94\x8c+/Users/sdreyer/Projects/embeddinghub/pkl.py\x94\x8c\x0bmy_function\x94h\x0bK\x04C\x08\x80\x00\xd8\x0b\x14\xd0\x04\x14\x94C\x00\x94))t\x94R\x94c__builtin__\n__main__\nh\x0bNNt\x94R\x94}\x94}\x94\x8c\x0f__annotations__\x94}\x94s\x86\x94b.'`

If I unpickle it, I get:
`<function my_function at 0x105486160>`

In python you can inspect this to get the string value, but in go you can't (as far as im aware). This means we need another field that can store it